src/Microsoft.DotNet.Interactive.ExtensionLab/NteractDataExplorerExtensions.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Text;
+using Microsoft.AspNetCore.Html;
+using Microsoft.DotNet.Interactive.Formatting;
+using Microsoft.DotNet.Interactive.Http;
+
+namespace Microsoft.DotNet.Interactive.ExtensionLab
+{
+    public static class NteractDataExplorerExtensions
+    {
+        public static TabularDataResourceExplorerView<NteractDataExplorer> ToView(this NteractDataExplorer explorer)
```

Comment:
Let's try to get rid of this method. It looks like it should just be an implementation detail of the formatter registrations.